### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by YuriySamorodov**
Fixed couple grammar typos

### CODE REVIEW

File: microsoft-365/security/defender/api-advanced-hunting.md [hunk 1]
```diff
@@ -48,11 +48,11 @@ The following conditions relate to all queries.
 6. A `429` HTTP response code indicates that you've reached a quota, either by number of requests sent, or by allotted running time. Read the response body to understand the limit you have reached. 
 
 > [!NOTE]
-> All quotas listed above (for example 15 calls per min) are per tenant size. These quotas are the minimum.
+> All quotas listed above (for example 15 calls per min) are tenant wide. These quotas are the minimum.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender/api-advanced-hunting.md:51]:
```suggestion
> All quotas listed above (for example 15 calls per min) are tenant wide. These quotas are the minimums.
```

File: microsoft-365/security/defender/api-advanced-hunting.md [hunk 2]
```diff
@@ -48,11 +48,11 @@ The following conditions relate to all queries.
...
 
 ## Permissions
 
-One of the following permissions is required to call the advanced hunting API. To learn more, including how to choose permissions, see [Access the Microsoft 365 Defender Protection APIs](api-access.md)
+One of the following permissions is required to call the Advanced Hunting API. To learn more, including how to choose permissions, see [Access the Microsoft 365 Defender Protection APIs](api-access.md)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender/api-advanced-hunting.md]:
```suggestion
One of the following permissions is required to call the advanced hunting API. To learn more, including how to choose permissions, see [Access the Microsoft 365 Defender Protection APIs](api-access.md).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @YuriySamorodov, kindly commit the minor additional changes. Many thanks.

File: microsoft-365/security/defender/api-advanced-hunting.md (hunk 2 above)
Comment (YuriySamorodov) [reply microsoft-365/security/defender/api-advanced-hunting.md]: Hey @JohanFreelancer9 ,
Just wondering, why Advanced Hunting should be in lower case? Isn't it a personal name of this API?

Comment (JohanFreelancer9) [reply microsoft-365/security/defender/api-advanced-hunting.md]: Hey @YuriySamorodov, thank you for the question. I know capitalization of this term might be somewhat inconsistent throughout the documentation, but the latest guidance is that this is a descriptive name, like many other capabilities, not a product name, and should not be capitalized. Thanks again.

Comment (YuriySamorodov) [reply microsoft-365/security/defender/api-advanced-hunting.md]: Thanks @JohanFreelancer9 , so in other words Advanced Hunting is not the specific API name?

Comment (JohanFreelancer9) [reply microsoft-365/security/defender/api-advanced-hunting.md]: @YuriySamorodov, that is correct, it is a descriptive name.

Comment (YuriySamorodov) [reply microsoft-365/security/defender/api-advanced-hunting.md]: Make sense now, thank you @JohanFreelancer9

**Pull request merged**